site/en/_partials/privacy-sandbox/ps-implementation-status.md

```diff
@@ -0,0 +1,5 @@
+- The [Topics API proposal](https://github.com/jkarlin/topics) has been through a [public discussion](https://github.com/jkarlin/topics/issues) phase  and is now available in an [origin trial](https://github.com/jkarlin/topics/issues). The API is now approaching launch for scaled adoption. 
+- The API needs your feedback. If you have comments, create an issue on the [Topics Explainer repository](https://github.com/jkarlin/topics) or participate in discussions in the [Improving Web Advertising Business Group](https://www.w3.org/community/web-adv/participants). The explainer has a number of [open questions](https://github.com/jkarlin/topics/issues) that still require further definition.
+- [The Privacy Sandbox timeline](http://privacysandbox.com/timeline) provides implementation timelines for the Topics API and other Privacy Sandbox proposals.
+
+[Topics API: latest updates](/docs/privacy-sandbox/topics/latest/) details changes and enhancements to the Topics API and implementations.
```

Comment:
Just not sure if this is a global statement. Leave or remove?

Reply:
It will be good to commit this when possible to avoid missing refs in upcoming topics docs PRs

Reply:
This can stay for now, though we may remove it later. However, it should become a bullet point.